### PR TITLE
SDK Integration of Inventory and Channel APIs

### PR DESCRIPTION
**Pull request by purquhart**

### CODE REVIEW

**Comment by nkuehn** [comment]
@ebonertz I'm not a useful reviewer for .NET SDK changes - @sshibani  and @jenschude  are the only ones that can give qualified feedback at the moment.

**Comment by ebonertz** [comment]
@jenschude @sshibani can you please review this PR so we can merge? Our partner is waiting for this new functionality

File: commercetools.NET/Channels/Channel.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            this.ReviewRatingStatistics = data.reviewRatingStatistics != null ? new ReviewRatingStatistics(data.reviewRatingStatistics) : null;
+            this.Custom = new CustomFields.CustomFields(data.custom);
+            if (data.geoLocation != null && data.geoLocation.type != null)
+            {
+                switch ((string)data.geoLocation.type)
+                {
+                    case "Point":
+                        this.GeoLocation = new Point(data.geoLocation);
+                        break;
+                    default:
```
[the remark this answers is not in the document]
Comment (purquhart) [reply commercetools.NET/Channels/Channel.cs]: Hi @jenschude, just to confirm your request - are you requesting that this:

                    case "Point":
                        this.GeoLocation = new Point(data.geoLocation);
                        break;
                    default:
                        throw new NotImplementedException(string.Format("Unhandled GeoLocation Type: {0}", data.geoLocation.type));

become this:

                    case "Point":
                        this.GeoLocation = new Point(data.geoLocation);
                        break;
                    default:
                        // leave GeoLocation null, the returned data contains an unhandled GeoLocation type
                        break;

[the remark this answers is not in the document]
Comment (jenschude) [reply commercetools.NET/Channels/Channel.cs]: Thought about it again. Better would be to create a GeoJson class with properties type and coordinates. Where coordinates just holds the dynamic data without any further deserialization.

And create a GeoJson instance in the default case

**Comment by nkuehn** [comment]
You could consider a native GeoCoordinate, too : https://msdn.microsoft.com/de-de/library/system.device.location.geocoordinate(v=vs.110).aspx

**Comment by jenschude** [comment]
I would only add a helper properties to the Point class to get/set a GeoCoordinate instance. In order to be compatible with the GeoJSON format.

**Comment by ebonertz** [comment]
@purquhart Do you have an update on the changes requested? Our client is waiting for this functionality so they can begin building a new integration

**Pull request merged**